stylus/components/nav.styl

```diff
@@ -62,7 +62,9 @@ $nav-item-icon
 
     :hover > &
         color var(--charcoalGrey)
-
+    +small-screen()
+        svg
+            display inline
```

Comment:
I don't understand what is going on here ? As a guess, the padding between the icon and the text comes from the line-height here?

Reply:
Woo, this is not exactly what I wanted to commit... I wanted to add this display inline here : https://github.com/cozy/cozy-ui/blob/7881ef467b8ba651ee654ade3346fe1a3bcf5880/stylus/components/nav.styl#L73 since we are already doing few stuff with the svg.

Reply:
Still I do not exactly understand how putting inline fixes the padding between the icon and the text, is it because of the line-height ?